lxd/network/driver_bridge.go

```diff
@@ -1457,9 +1457,12 @@ func (n *bridge) setup(oldConfig map[string]string) error {
 				Link: ip.Link{Name: tunName},
 			}
 
+			remotes := strings.Split(tunRemote, ",")
```

Comment:
Please can you use `shared.SplitNTrimSpace(args[1], ",", -1, false)` here.

Reply:
Done!